src/components/DataInputForm.tsx

```diff
@@ -0,0 +1,85 @@
+
+import React, { useState } from 'react';
+import { View, TextInput, Button, Alert } from 'react-native';
+import { DataInputRequest, User } from '../types/Types';
+import { postDataInput } from '../apis/DataInputApi';
+
+const DataInputForm: React.FC = () => {
+  const [user, setUser] = useState<User>({
+    userId: Math.random().toString(36).substring(7), // Set a random userId before submitting
+    firstName: '',
+    lastName: '',
+    email: '',
+    phoneNumber: '',
+    address: '',
+  });
+
+  const handleDataInput = async () => {
+    try {
+      const requestData: DataInputRequest = {
+        user: user,
+      };
```

Comment:
`DataInputRequest` should just be same as `User`

This is the input payload to DataInputApi:
```
{
  "userId": "asdfasdf",
  "firstName": "vaibhav",
  "lastName": "vasant",
  "email": "v12@gmail.com",
  "phoneNumber": "7633824597",
  "address": "None"
}
```